src/core/components/states/stateSkeletonBar/stateSkeletonBar.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { StateSkeletonBar } from './stateSkeletonBar';
+
+describe('<StateSkeletonBar /> component', () => {
+    const createTestComponent = (props?: Partial<React.ComponentPropsWithRef<typeof StateSkeletonBar>>) => {
```

Comment:
Can't we just do:
```
Partial<IStateSkeletonBarProps>
```

Reply:
Not if we want to test for the forwarded ref unfortunately. That specific test isn't very useful though, at least not for this component. What do you think?

Reply:
I think we can remove the `ref` test to be aligned with all the other components